manifests/postgres.yaml

```diff
@@ -46,12 +46,12 @@ spec:
           initialDelaySeconds: 60
           timeoutSeconds: 5
           failureThreshold: 6
-        readinessProbe:
+        readiness-probe:
```

Comment:
I have a good reason (at least to me) to use spinal-case in mystack yaml files. 
When an app is defined, for example, we choose its name. Kubernetes doesn't allow camel case, so there can't be a receiptsVerifier, only a receipts-verifier. 
In that case, the app names would be spinal-case and the variables would be camel case (like readinessProbe). 

Do you still prefer camel case? I can change it back if you do.